submissions/SerhiiYakovenko/html-css-popup/css/style.css

```diff
@@ -0,0 +1,178 @@
+* {
+  margin: 0;
+  padding: 0;
+  color: black;
+}
+
+body {
+  font-family: "Orbitron", sans-serif;
+  background: url(".././img/background.jpg") no-repeat center center fixed;
+  background-size: cover;
+  height: 100%;
+  overflow: hidden;
+}
+
+img {
+  max-width: 100%;
+  max-height: 40px;
+}
+
+.release {
+  font-size: 35px;
+  color: yellow;
+  background-color: black;
+  position: fixed;
+  top: 30%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  text-align: center;
+  opacity: 0.8;
+}
+
+.nav {
+  margin: 10px;
+  float: right;
+  display: flex;
+  align-items: center;
+}
+
+.menu {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+}
+
+.nav-item {
+  list-style: none;
+  margin: 10px 5px;
+}
+
+.popup-item {
+  padding: 10px;
+}
+
+.kottans:focus,
+.nav-item:focus,
+.input:focus + label,
+.menu-item:focus,
+input:focus + label,
+.popup-item:focus,
+.menu-item:focus {
+  outline: none;
+  border: 1.5px inset gray;
+}
+
+.kottans:hover,
+.popup-item:hover,
+.more-label:hover {
+  cursor: pointer;
+  outline: none;
+  border: 1.5px inset gray;
+}
+
+.menu__popup {
+  display: block;
+  padding: 8px;
+}
+
+.nav-item:hover {
+  cursor: pointer;
+}
+
+.popup-input,
+.more-input {
+  position: absolute;
+  z-index: -1;
+  opacity: 0;
+}
+
+.popup-label {
+  padding: 8px;
+  position: relative;
+  top: -25px;
+  display: block;
+  width: 150px;
+  height: 30px;
+  cursor: pointer;
+  background: url(".././img/logo.svg") no-repeat center;
+}
+
+.xbox,
+.ps4 {
+  position: relative;
+  top: -15px;
+}
+
+.main {
+  position: relative;
+}
+
+.popup-main {
+  justify-content: center;
+  display: none;
+  flex-wrap: wrap;
+  position: absolute;
+  right: -50%;
+  margin-top: 20px;
+  align-self: flex-end;
+  width: 350px;
+  height: 280px;
+  padding: 10px;
+  overflow-y: scroll;
+  z-index: 1;
+  border: 2px inset black;
+  background-color: white;
+  opacity: 0.7;
+}
+
+.popup-item {
```

Comment:
To prevent the elements from moving when hovering, you can set them transparent frames.
<img width="1370" alt="Снимок экрана 2020-10-27 в 10 10 25 AM" src="https://user-images.githubusercontent.com/14906260/97274122-e1c7bc80-183c-11eb-8551-195456c4dd29.png">

Reply:
thx 👍